src/components/PlayerDash.jsx

```diff
@@ -15,38 +17,47 @@ export const PlayerDash = React.createClass({
   getInitialState() {
     return {
       players: [],
+      graphdata: [],
       loaded: false
     }
   },
 
   componentWillMount() {
+    this.firebase = new FirebaseLib();
     // We are likely to want to read all player data e.g. top adversary, last 10 matches etc
     // So leaving this here for now. Woould be nicer to have a store where this data is kept without requesting again
-    let fbPath = [conf.firebaseUrl, 'players'].join('/');
-    this.fireBase = new Firebase(fbPath);
     this.loadData(); // should update to bindAsObject/Array
   },
 
   render() {
```

Comment:
Using `the` as a variable prefix is a pet hate of mine, I see it everywhere and have definitely been guilty of it myself. I find it so redundant and adds nothing to code context. Also, if my comments ever sound negative they're never meant to be - feel free to reply that `the` is an awesome prefix and I'm simply wrong :smiley_cat: 

I'd also add destructuring below to remove the `this.state` cruft out of the templates.

```js
let { players, graphdata } = this.state;
// I would capitalise the d too -> graphData

const player = _.find(players, p => p.id == this.props.params.playerId);

//...

{ graphdata &&
        <div className="col-md-9">
```